### DISCUSSION TITLE
Comitup hotspot password

### DISCUSSION THREAD

**Discussion by rkosick** [outdoorbits/little-backup-box#195] [Ideas]
Have you thought about an option to set up a password for the Comitup hotspot? It's would be a line in /etc/comitup.conf like:

ap_password: your_custom_password

I tried it before and it worked. Like you said, it would be removed after an update and it did. I was just thinking about it: Your are not near a known wifi and backup lots of data and lbb is always available as an unsecured hotspot. I don't think there is any issue with that, since it is password protected. But maybe somebody has concerns about it.

I was thinking about the minidlna server. But I did not understand how it works. I did a test with generating thumbnails and the minidlna server never shared any files. That's good for my case. I don't want to be connected to a public access point and have a minidlna server running for everybody. My concern was that would happen with a comitup hotspot without any password to. But for me it did't.

**Comment by outdoorbits** [comment]
Both very good points. minidlna was already included when I came to the project. Everybody: Does anybody use minidlna??? I would suggest, just to remove it. Other opinions?

Using the password for comitup too seems to be a very good idea. I will implement this within the next few days.

Thank you for your good ideas and excellent preparatory work!

**Comment by ruebyi** [comment]
Never used minidlna- I understand what was the intention, but I don’t think it’s really important today. 
mit doesn’t seem like dlna is a huge thing anymore.

**Comment by rkosick** [comment]
I never used minidlna with lbb and like I said, I even don't know how activate it. Just tried to see if there is a porblem about it. Didn't found any settings and its not working at all for me. It was just a concern for me as I don't want anybody to have access to my files.

[Edit]: I guess there could be someone who would like to use it. Like always. But probably 1 in a thousand? I for myselfe would love to have as much options as possible everytime. But I have to be able to use the device without accidental activate something I don't want to. The minidlna server only makes sense in private networks.

Keep going. Thanks a lot for your effort.

**Reply by outdoorbits** [reply]
Both is uploaded. Thank you for testing!

**Reply by rkosick** [reply]
Not working right now for me. I did an update and deleted the comitup credentials. After that the hotspot was still without a password protection. Update seemed to work as I saw how minidlna was removed. Also after connecting via ssh, I can't see the password in /etc/comitup.conf

**Reply by rkosick** [reply]
Maybe I have to set up the global password again?

**Reply by rkosick** [reply]
After setting the global password again, it worked. Good idea to not make an extra setting so setup the hotspot password and just make it the global password.

**Reply by rkosick** [reply]
On a fresh install, is it also the default password?

**Reply by rkosick** [reply]
Is there a default password anyway? Can't remember.
And one thing: Minimum password lenght for wifi wpa2 is at least 8 characters? Should be mentioned while setting up the password in the settings. Or has to be mantatory.

**Comment by outdoorbits** [comment]
The requested password length should be printed in settings, global password, isn't it?

**Reply by rkosick** [reply]
Oh my mistake. It is printed there. Perfect.

**Comment by outdoorbits** [comment]
On a fresh install the is no password set. Good point to set password during update.

**Reply by rkosick** [reply]
That would be good. I figured it out but who knows.
Looks great then for me. I don't have any ideas how to make it better anymore. Thank's a lot for your work.
